lark/lark.py

```diff
@@ -1,6 +1,7 @@
 from abc import ABC, abstractmethod
 import sys, os, pickle, hashlib
 import tempfile
+import types
```

Comment:
Is this needed?

Reply:
Yes, to type `re_module: types.ModuleType` on line 258. Otherwise `re_module` doesn't support being assigned two different Modules.